generate-protos.sh

```diff
@@ -15,17 +15,23 @@ protoc -I="$PROTOS" \
     "$PROTOS"/service.proto \
     "$PROTOS"/model_registry.proto \
     "$PROTOS"/databricks_artifacts.proto \
+    "$PROTOS"/databricks_model_artifacts.proto \
     "$PROTOS"/scalapb/scalapb.proto
 
 OLD_SCALAPB="from scalapb import scalapb_pb2 as scalapb_dot_scalapb__pb2"
 NEW_SCALAPB="from .scalapb import scalapb_pb2 as scalapb_dot_scalapb__pb2"
-sed -i'.old' -e "s/$OLD_SCALAPB/$NEW_SCALAPB/g" "$PROTOS/databricks_pb2.py" "$PROTOS/service_pb2.py" "$PROTOS/model_registry_pb2.py" "$PROTOS/databricks_artifacts_pb2.py"
+sed -i'.old' -e "s/$OLD_SCALAPB/$NEW_SCALAPB/g" "$PROTOS/databricks_pb2.py" "$PROTOS/service_pb2.py" "$PROTOS/model_registry_pb2.py" "$PROTOS/databricks_artifacts_pb2.py" "$PROTOS/databricks_model_artifacts_pb2.py"
 
 OLD_DATABRICKS="import databricks_pb2 as databricks__pb2"
 NEW_DATABRICKS="from . import databricks_pb2 as databricks__pb2"
-sed -i'.old' -e "s/$OLD_DATABRICKS/$NEW_DATABRICKS/g" "$PROTOS/service_pb2.py" "$PROTOS/model_registry_pb2.py" "$PROTOS/databricks_artifacts_pb2.py"
+sed -i'.old' -e "s/$OLD_DATABRICKS/$NEW_DATABRICKS/g" "$PROTOS/service_pb2.py" "$PROTOS/model_registry_pb2.py" "$PROTOS/databricks_artifacts_pb2.py" "$PROTOS/databricks_model_artifacts_pb2.py"
+
+OLD_SERVICE="import service_pb2 as service__pb2"
```

Comment:
What does this do?

Reply:
Great question, I kinda took it for granted until you asked.
The essence of why we do this comes down to : https://github.com/protocolbuffers/protobuf/issues/1491
TLDR: https://github.com/protocolbuffers/protobuf/issues/1491#issuecomment-238123573
Thus we need to convert the relative imports into absolute import.